app/src/processing/app/Base.java

```diff
@@ -195,9 +194,20 @@ public Base(String[] args) throws Exception {
 
     BaseNoGui.initPortableFolder();
 
+    // Look for a possible "--preferences-file" parameter and load preferences
     BaseNoGui.initParameters(args);
 
-    splashScreenHelper.splashText(tr("Loading configuration..."));
+    CommandlineParser parser = new CommandlineParser(args);
+    parser.parseArgumentsPhase1();
```

Comment:
Right. Sounds like the commandline parsing needs another refactor after this one then (or, at least a `parseArgumentsPhase0` to just parse the preferences path).